app/views/exam_templates/_student_info.html.erb

```diff
@@ -24,9 +24,17 @@
 
   <div id="exam-cover-display-<%= exam_template.id %>" class="<%= exam_template.automatic_parsing ? 'flex-display' : 'no-display' %>">
     <div id="exam_cover-<%= exam_template.id %>">
-      <img src="<%= show_cover_course_exam_template_path(@current_course, exam_template) %>" class="crop-target"
-           alt="<%= t(:'exam_templates.parsing.cover_page_for', id: exam_template.id) %>"
-           loading="lazy">
+      <div class="exam-crop-container">
```

Comment:
It seems to me like this div can be merged into the outer one (`id="exam_cover-..."`), since I don't think the outer one is doing much as it is and you've only added one child do it